tensorboardX/writer.py

```diff
@@ -226,6 +227,9 @@ def __init__(
             filename_suffix: Optional[str] = '',
             write_to_disk: Optional[bool] = True,
             log_dir: Optional[str] = None,
+            comet: Optional[bool] = False,
```

Comment:
The example (pytorch-rich-logging-example.py) in comet-ml needs `api_key` as well. How one pass this parameter here?

Reply:
Got this: `COMET INFO: Comet API key is not set.` I have comet-ml==3.9.1. Did I misses any requirements?

Reply:
You can set your Comet API key in four ways:

1. Ammad is adding a parameter here (should be done today)
2. You can also use an env variable, COMET_API_KEY
3. You can setup a ~/.comet.config file
4. You can set in in python code at top (os.environ["COMET_API_KEY"] = ...)

The first way will probably be easiest to get started. More information here:

https://www.comet.ml/docs/python-sdk/advanced/#python-configuration

Reply:
FYI, Even if I passed the correct API key and got `COMET INFO: Experiment is live on comet.ml https://www.comet.ml/...` (The experiment should have been uploaded correctly), The terminal still says: `COMET INFO: Comet API key is not set. You'll have to provide it explictly to experiments and API`.